src/large_pages/node_large_page.cc

```diff
@@ -369,16 +422,19 @@ int MapStaticCodeToLargePages() {
     return MoveTextRegionToLargePages(r);
 
   return -1;
-#elif defined(__FreeBSD__)
+#elif defined(__FreeBSD__) || defined(__APPLE__)
   return MoveTextRegionToLargePages(r);
 #endif
 }
 
 bool IsLargePagesEnabled() {
 #if defined(__linux__)
   return IsTransparentHugePagesEnabled();
-#else
+#elif defined(__FreeBSD__)
   return IsSuperPagesEnabled();
+#elif defined(__APPLE__)
+  // pse-36 flag is present in recent mac x64 products.
```

Comment:
How recent is recent? Node.js supports macOS 10.11 and up.

Reply:
Then it is fine El Capitan is recent enough :)